nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -608,37 +612,69 @@ define([
             const meta = cell.metadata;
             meta.kbase.attributes.lastLoaded = new Date().toUTCString();
             cell.metadata = meta;
-            render().then(() => {
-                jobManager.addHandler('modelUpdate', {
-                    execMessage: (jobManagerContext) => {
-                        // Update the execMessage panel with details of the active jobs
-                        controlPanel.setExecMessage(
-                            Jobs.createCombinedJobState(
+            render()
```

Comment:
Turns out this chunk where we render, then do a global parameter check to update the state with is actually usefully important. Updated this to revert that change, merge it with other recent changes, and introduce the flow that looks up missing files before validating that things are as correct as they should be.